StarRailBattleSim/src/relics/relics/EagleOfTwilightLine.java

```diff
@@ -0,0 +1,35 @@
+package relics.relics;
+
+import battleLogic.Battle;
+import characters.AbstractCharacter;
+import enemies.AbstractEnemy;
+import powers.PermPower;
+import powers.PowerStat;
+import relics.AbstractRelicSetBonus;
+
+import java.util.ArrayList;
+
+public class EagleOfTwilightLine extends AbstractRelicSetBonus {
+    public EagleOfTwilightLine(AbstractCharacter owner, boolean fullSet) {
+        super(owner, fullSet);
+    }
+
+    public EagleOfTwilightLine(AbstractCharacter owner) {
+        super(owner);
+    }
+
+    @Override
+    public void onEquip() {
+        if (this.owner.elementType == AbstractCharacter.ElementType.WIND) {
+            this.owner.addPower(PermPower.create(PowerStat.SAME_ELEMENT_DAMAGE_BONUS, 10, "Genius of Brilliant Stars Quantum bonus"));
+        }
+    }
+
+    @Override
+    public void afterAttackFinish(AbstractCharacter character, ArrayList<AbstractEnemy> enemiesHit, ArrayList<AbstractCharacter.DamageType> types) {
```

Comment:
shouldn't this be using `onUseUltimate`? the set bonus works even if it doesn't attack an enemy with the ultimate.